service/src/main/resources/api/service_openapi.yaml

```diff
@@ -2052,6 +2130,15 @@ components:
         azureDisk:
           $ref: '#/components/schemas/AzureDiskCreationParameters'
 
+    CreateControlledAzureVmRequestBody:
+      description: Payload for requesting a new controlled Azure VM resource.
+      type: object
+      properties:
+        common:
+          $ref: '#/components/schemas/ControlledResourceCommonFields'
+        azureVm:
+          $ref: '#/components/schemas/AzureVmCreationParameters'
+
```

Comment:
needs a `jobControl` entry passed in. Like here [L2168 of main](https://github.com/DataBiosphere/terra-workspace-manager/blob/174e3727b6a03fcd189e14f1adf2219ae0e23149/service/src/main/resources/api/service_openapi.yaml#L2168)